signer/src/storage/postgres.rs

```diff
@@ -1467,10 +1475,10 @@ impl super::DbRead for PgStore {
               , signer_set_public_keys
               , signature_share_threshold
             FROM sbtc_signer.dkg_shares
-            WHERE aggregate_key = $1;
+            WHERE substring(aggregate_key FROM 2) = $1;
```

Comment:
Just a nit -- I think I would have preferred that we stored this in an extra column instead of messing around with it at query-time. I think we do something similar and even more convoluted somewhere else..